internal/framework/events/loop.go

```diff
@@ -71,7 +71,7 @@ func (el *EventLoop) Start(ctx context.Context) error {
 
 			batchLogger.Info("Handling events from the batch", "total", len(batch))
 
-			el.handler.HandleEventBatch(ctx, batchLogger, batch)
+			el.handler.HandleEventBatch(ctx, batch)
```

Comment:
what's the rational for this removal? 
this was done, so that we group messages related to handling a specific batch together -- they will use the same batchID.
```go
batchLogger := el.logger.WithName("eventHandler").WithValues("batchID", el.currentBatchID)
```
it also followed https://github.com/nginxinc/nginx-gateway-fabric/blob/main/docs/developer/logging-guidelines.md#logger-injection

Reply:
The logger was added to the event handler object to avoid passing it down an entire function call tree. Though it may be simpler now, I can probably just pass this down the tree as before to preserve the batch ID.